templates/publisher/account-snaps.html

```diff
@@ -159,15 +163,16 @@ <h3 class="p-heading--five">Join the community</h3>
                   {% endif %}
                 </td>
                 {% if published %}
-                  <td>{{ snaps[snap].latest_revisions[0].channels[0] }}</td>
+                  <td>{{ snaps[snap].latest_revision.channels[0]|title }}</td>
```

Comment:
I'm not sure if we want to capitalise channel names. They are kind of part of snapcraft brand and are usually lower case ("edge" instead of "Edge").

Thoughts @therealjuan ?